TRANSFORMERS/CALCULUS/DOUBLE_INDEFINITE_INTEGRAL/DOUBLE_INDEFINITE_INTEGRAL.py

```diff
@@ -41,10 +42,10 @@ def contains_only_numbers(column):
             raise ValueError(
```

Comment:
Instead of repeating the same thing three times, we can define a function for this